R/recipe.R

```diff
@@ -129,10 +126,22 @@ recipe.data.frame <-
     }
 
     if (any(table(vars) > 1)) {
-      rlang::abort("`vars` should have unique members")
+      offenders <- vctrs::vec_count(vars)
+      offenders <- offenders$key[offenders$count != 1]
+
+      cli::cli_abort(c(
+        x = "{.arg vars} must have unique members.",
```

Comment:
What is a member?

Reply:
should read "values". and it does now in [update snapshots](https://github.com/tidymodels/recipes/pull/1251/commits/320b55d18976cbb97871deb5be7829ad3625970b)